third_party/googleapis/README.grpc-gateway

```diff
@@ -3,14 +3,15 @@ Google APIs
 
 Project: Google APIs
 URL: https://github.com/google/googleapis
-Revision: 3544ab16c3342d790b00764251e348705991ea4b
+Revision: ca1372c6d7bcb199638ebfdb40d2b2660bab7b88
 License: Apache License 2.0
 
 
 Imported Files
 ---------------
 
 - google/api/annotations.proto
+- google/api/field_behavior.proto
```

Comment:
@johanbrandhorst This is the "addition to the README" I was referencing. It doesn't seem like the most essential file but I came across it while searching for any guidance on adding files to `third_party`